juliadoc/jldoctest.py

```diff
@@ -808,6 +808,7 @@ def test_group(self, group, filename):
         j = Popen(["../julia"], stdin=PIPE, stdout=PIPE, stderr=STDOUT)
         j.stdin.write("macro raw_str(s) s end;nothing\n".encode('utf-8'))
         j.stdin.write("_ans = nothing\n".encode('utf-8'))
+        j.stdin.write("pushdisplay(TextDisplay(IOContext(IOContext(STDOUT, :multiline => true), :limit => true))); nothing\n".encode('utf-8'))
```

Comment:
I guess this now becomes 0.5-only. So if any bugfixes are ever needed to JuliaDoc for 0.4 we can go back and branch.